keps/sig-cloud-provider/20190125-out-of-tree-ibm.md

```diff
@@ -40,60 +43,135 @@ status: provisional
 
 - [X] k/enhancements issue in release milestone and linked to KEP (https://github.com/kubernetes/enhancements/issues/671)
 - [ ] KEP approvers have set the KEP status to `implementable`
-- [ ] Design details are appropriately documentedbs
+- [ ] Design details are appropriately documented
 - [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [ ] Graduation criteria is in place
 - [ ] "Implementation History" section is up-to-date for milestone
 - [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
 
+## Terms
+
+- **CCM:** Cloud Controller Manager - The controller manager responsible for
```

Comment:
Suggest to provide hyperlink to the Kubernetes doc page for CCM.

Reply:
Will do